out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -4,6 +4,53 @@
     }
 </style>
 
+<script type="text/javascript">
+  function testPaymentMethodCredentials() {
+    // the DOM element the test result will be rendered to
+    var resultSpan = document.getElementById('test_credentials_result');
+
+    // clear any previous test results
+    resultSpan.innerHTML = '';
+
+    var bodyParams = {
+      apiUrl: document.getElementById('configApiUrl').value,
+      httpUser: document.getElementById('configHttpUser').value,
+      httpPass: document.getElementById('configHttpPass').value
+    };
+
+    var successText = '[{oxmultilang ident="success_credentials"}]';
+    var failureText = '[{oxmultilang ident="error_credentials"}]';
+
+    var requestUrl = '[{ $oViewConf->getAjaxLink() }]cmpid=container&container=payment_main&fnc=checkPaymentMethodCredentials';
+
+    var paramString = Object.keys(bodyParams).map((key) => {
```

Comment:
Let's avoid arrow functions for broader compatibility. We don't really have a clear browser compatibility matrix.

Reply:
thanks, replaced the arrow function